scripts/simtest/cargo-simtest

```diff
@@ -96,6 +96,11 @@ fi
 # Must supply a new temp dir - the test is deterministic and can't choose one randomly itself.
 export TMPDIR=$(mktemp -d)
 
+# Set the example move package for the simtest static initializer
+# https://github.com/MystenLabs/sui/blob/7bc276d534c6c758ac2cfefe96431c2b1318ca01/crates/sui-proc-macros/src/lib.rs#L52
+root_dir=$(git rev-parse --show-toplevel)
+export SIMTEST_STATIC_INIT_MOVE=$root_dir"/examples/move/basics"
```

Comment:
Suggestion: Put these into quotes to prevent potential issues with white spaces, etc.
```suggestion
root_dir="$(git rev-parse --show-toplevel)"
export SIMTEST_STATIC_INIT_MOVE="$root_dir/examples/move/basics"
```